packages/governance/src/contractGovernor.js

```diff
@@ -90,10 +91,15 @@ const validateQuestionFromCounter = async (zoe, electorate, voteCounter) => {
  * up to the contract.
  *
  * The contractGovernor creatorFacet includes voteOnParamChanges(),
- * which is used to create questions that will automatically update
- * contract parameters if passed. This facet will usually be closely held. The
- * creatorFacet can also be used to retrieve the governed instance, publicFacet,
- * and its creatorFacet with voteOnParamChanges() omitted.
+ * voteOnFilter, and voteOnApiInvocation. voteOnParamChanges() is used to create
```

Comment:
inconsistent use of '()'. Consider dropping the parens and using backticks.
> The contractGovernor creatorFacet includes `voteOnParamChanges`, `voteOnFilter`,
> and `voteOnApiInvocation`. `voteOnParamChanges` is used to create questions that

Reply:
alternately, refer to them subsequently by the unique string:
inconsistent use of '()'. Consider dropping the parens and using backticks.
> The contractGovernor creatorFacet includes `voteOnParamChanges`, `voteOnFilter`,
> and `voteOnApiInvocation`. ParamChanges is used to create questions that

Reply:
done